docs/topics/native/native-spm.md

```diff
@@ -80,26 +80,58 @@ To set up the publishing of an XCFramework:
    
-   `./gradlew :shared:assembleSharedReleaseXCFramework`
+   ```shell
+   ./gradlew :shared:assembleSharedXCFramework
+   ```
   
    The resulting framework will be created as the `shared/build/XCFrameworks/release/Shared.xcframework` folder in your project directory.
 
    > In case you work with a Compose Multiplatform project, use the following Gradle task:
    >
-   > `./gradlew :composeApp:assembleSharedReleaseXCFramework`
+   > ```shell
+   > ./gradlew :composeApp:assembleSharedXCFramework
+   > ```
    >
    > You can then find the resulting framework in the `composeApp/build/XCFrameworks/release/Shared.xcframework` folder.
    >
    {type="tip"}
 
 ### Prepare the XCFramework and the Swift package manifest
 
-1. Put the `Shared.xcframework` folder in a ZIP archive and calculate the checksum for the resulting archive, for example:
+1. Compress the `Shared.xcframework` folder in a ZIP file and calculate the checksum for the resulting archive, for example:
    
    `swift package compute-checksum Shared.xcframework.zip`
 
-2. <anchor name="upload"></anchor> Upload the ZIP file to the file storage of your choice.
-3. Choose any directory and locally create a `Package.swift` file with the following code:
+2. <anchor name="upload"></anchor> Upload the ZIP file to the file storage of your choice. The file should be accessible
+   by a direct link. For example, here's how you can do it using releases in GitHub:
+   
+   <deflist collapsible="true">
+       <def title="Upload to a GitHub release">
+           <list type="decimal">
+               <li>Go to <a href="https://github.com">GitHub</a> and log in to your account.</li>
+               <li>Navigate to the repository where you want to create a release.</li>
+               <li>In the <b>Releases</b> section on the right, click the <b>Create a new release</b> link.</li>
+               <li>Fill in the release information, add or create a new tag, specify the release title and write a description.</li>
+               <li>
+                   <p>Upload the ZIP file with the XCFramework through the <b>Attach binaries by dropping them here or selecting them</b> option at the bottom:</p>
+                   <img src="github-release-description.png" alt="Fill in the release information" width="700"/>
+               </li>
+               <li>Click <b>Publish release</b>.</li>
+               <li>
+                   <p>Under the <b>Assets</b> section of the release, right-click on the ZIP file and select <b>Copy link address</b> or use a similar option in your browser:</p>
+                   <img src="github-release-link.png" alt="Copy the link to the uploaded file" width="500"/>
+               </li>
+         </list>
+       </def>
+   </deflist>
+
+3. [Recommended] Check that the link works and that the file can be downloaded. In the terminal, run the following command:
+
+    ```shell
+    curl <downloadable link to the uploaded XCFramework ZIP file>
```

Comment:
The `shell` language here highlights the "link" within the brackets, not sure if that is better than no highlight at all.